data/styles/adwaita.mplstyle

```diff
@@ -0,0 +1,61 @@
+# adwaita
+patch.facecolor: FAFAFA
+patch.linewidth: .3
+
+axes.axisbelow: True
+axes.edgecolor: black
+axes.facecolor: white
+axes.grid: True
+axes.labelcolor: black
+axes.labelsize: 11
+axes.linewidth: .8
+axes.prop_cycle: cycler('color', ['4C72B0', '55A868', 'C44E52', '8172B2', 'CCB974', '64B5CD'])
+axes.titlesize: 14
+
+font.family: sans-serif
+font.sans-serif: Cantarell
+font.weight: normal
+font.size: 12
```

Comment:
Or these... 
Basically the fonts family, sans-serif, weight and size properties can all be set in the config file. For some reason the stylesheet seem to override at least the font sizes that are chosen. Making it impossible to change the font size.